bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/t23t23_Incl_tPythia_narrow_M700/t23t23_Incl_tPythia_narrow_M700_proc_card.dat

```diff
@@ -0,0 +1,16 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define l+ = e+ mu+
+define l- = e- mu-
```

Comment:
can you please cleanup the datacards according to https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T
(i.e. remove multiple model import, remove lepton and neutrino definitions if they are standard and are not used, etc)